web/src/angular/package.json

```diff
@@ -4,6 +4,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
+    "dev": "ng serve",
```

Comment:
Identical to start script?

Reply:
Just added this as an alternative to start out of convenience, since in other frameworks "dev" is used. I can remove it, but this should'nt cause any problems.